OmiseSDK/Omise3DSViewController.swift

```diff
@@ -0,0 +1,131 @@
+import Foundation
+import WebKit
+
+
+/// Delegate to receive 3DS verification events.
+public protocol Omise3DSViewControllerDelegate: class {
+    
+    func omise3DSViewController(_ viewController: Omise3DSViewController, didFinish3DSProcessWithRedirectedURL redirectedURL: URL?)
```

Comment:
Oh I forgot to re-evaluate this because I thought that the failed case also call this method (You can tell from the Optional type)